File: src/main/java/com/example/chatService/repository/RoomBanRepository.java
package com.example.chatService.repository;

import com.example.chatService.entity.RoomBan;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface RoomBanRepository extends JpaRepository<RoomBan, Long> {
    boolean existsByRoomIdAndUserId(String roomId, String userId);
}

File: src/main/java/com/example/chatService/repository/RoomBanRepository.kt
package com.example.chatService.repository

import com.example.chatService.entity.RoomBan
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface RoomBanRepository : JpaRepository<RoomBan, Long> {
    fun existsByRoomIdAndUserId(roomId: String, userId: String): Boolean
}
